Helper/Order.php

```diff
@@ -312,6 +317,9 @@ public function __construct(
      */
     public function fetchTransactionInfo($reference, $storeId = null)
     {
+        if (isset($this->transactionInfo[$reference][$storeId])) {
+            return $this->transactionInfo[$reference][$storeId];
```

Comment:
Isn't the $reference key enough? Do we really need $storeId here? Seems superfluous.

Reply:
Yes, you are right. Even if down the road we handle different $storeId in one script, $storeId will be identified by $reference.
Fixed.